### PR TITLE
Add marketplaces field in Dropbox event collector

### PR DESCRIPTION
**Pull request by yaakovpraisler**
## Status
- [x] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

### CODE REVIEW

File: Packs/Dropbox/Integrations/DropboxEventCollector/DropboxEventCollector.yml [hunk 1]
```diff
@@ -110,6 +110,8 @@ script:
   subtype: python3
   type: python
   isfetchevents: true
+marketplaces:
+- marketplacev2
```
Comment (anara123) [inline Packs/Dropbox/Integrations/DropboxEventCollector/DropboxEventCollector.yml:114]: @yaakovpraisler you are missing release notes

**Review by JasBeilin — APPROVED** [review]

File: Packs/Dropbox/ReleaseNotes/1_0_5.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Dropbox Event Collector
+- Updated the Docker image to: *demisto/fastapi:1.0.0.36992*.
+- Added the "marketplaces" key with the "markeplacev2" value.
```
Comment (dorschw) [inline Packs/Dropbox/ReleaseNotes/1_0_5.md:5]:
```suggestion
- Documentation and metadata improvements.
```

https://xsoar.pan.dev/docs/documentation/release-notes#general-changes

**Review by dorschw — APPROVED** [review]

File: Packs/Dropbox/ReleaseNotes/1_0_5.md [hunk 3]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Dropbox Event Collector
+- Updated the Docker image to: *demisto/fastapi:1.0.0.36992*.
+- Documentation and metadata improvements.
```
Comment (tkatzir) [inline Packs/Dropbox/ReleaseNotes/1_0_5.md:5]: I'm not sure this line is needed.

Comment (yaakovpraisler) [reply Packs/Dropbox/ReleaseNotes/1_0_5.md]: Do you have another suggestion? we can't leave it empty...

Comment (dorschw) [reply Packs/Dropbox/ReleaseNotes/1_0_5.md]: second line is [standard](https://xsoar.pan.dev/docs/documentation/release-notes#general-changes) when there are no changes, we could either leave it or remove it and only leave the first one, of the docker image update.

Comment (tkatzir) [reply Packs/Dropbox/ReleaseNotes/1_0_5.md]: Leaving just the Docker image update.

**Review by tkatzir — APPROVED** [review]

**Pull request merged**